source/guide_ackee.rst

```diff
@@ -34,47 +32,81 @@ Ackee_ is released under the `MIT License`_.
-  /home/isabell/bin/yarnpkg -> /home/isabell/lib/node_modules/yarn/bin/yarn.js
-  + yarn@1.22.4
-  installed 1 package in 1.002s
   [isabell@stardust ~]$ 
 
-Installation
-============
+Clone the `GitHub repository <https://github.com/electerious/Ackee>`_ and install the dependencies:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone https://github.com/electerious/Ackee
+  [isabell@stardust ~]$ cd Ackee
+  [isabell@stardust Ackee]$ yarn
+
+Configuration
+=============
+
+Ackee Config
+------------
 
-Clone the `GitHub repository of Ackee`_:
+Create a new, empty ``.env`` file in the root of the Ackee_ directory. Add the following content, replacing the placeholders. The environment variables ``ACKEE_USERNAME`` and ``ACKEE_PASSWORD`` define the credentials to log into the web interface. If you want to use Ackee_ to analyze a domain other than ``isabell.uber.space``, you have to add it under ``ACKEE_ALLOW_ORIGIN`` (read more about the CORS headers `in the Ackee documentation <https://github.com/electerious/Ackee/blob/master/docs/CORS%20headers.md>`_). 
 
 .. code-block:: console
 
-  [isabell@stardust ~]$ git clone https://github.com/electerious/Ackee ~/ackee
-  Cloning into '/home/isabell/ackee'...
-  remote: Enumerating objects: 1528, done.
-  remote: Counting objects: 100% (1528/1528), done.
-  remote: Compressing objects: 100% (884/884), done.
-  remote: Total 7991 (delta 1016), reused 1083 (delta 641), pack-reused 6463
-  Receiving objects: 100% (7991/7991), 1.97 MiB | 3.95 MiB/s, done.
-  Resolving deltas: 100% (5471/5471), done.
-  [isabell@stardust ~]$ 
+  ACKEE_MONGODB=mongodb://<admin>_mongoroot:<password>@localhost:27017/admin
+  ACKEE_USERNAME=<ackee_username>
+  ACKEE_PASSWORD=<ackee_password>
+  ACKEE_ALLOW_ORIGIN="https://my-domain-other-than-isabell-on-stardust.de"
+
+
+Web Backend Config
+------------------
+
+.. note::
+
+    Ackee_ is running on port 3000.
+
+.. include:: includes/web-backend.rst
+
+Supervisord Daemon Setup
+------------------------
+
+Create ``~/etc/services.d/ackee.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:ackee]
+  directory=%(ENV_HOME)s/Ackee
+  command=yarn start
+  autostart=yes
+  autorestart=yes
+
+.. include:: includes/supervisord.rst
+
+Finishing installation
+======================
+
+Go to ``https://isabell.uber.space``. Login using the credentials defined in section `Ackee Config`. Add the domains you want to analyze in the web interface: After adding, it shows a tracking snippet you can add to these pages. If data does not show up after implementing the tracking script, remember that cross site tracking always takes extra consideration of `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`_.
 
 .. _Ackee: https://ackee.electerious.com/
-.. _GitHub repository of Ackee: https://github.com/electerious/Ackee
+.. _MongoDB: https://mongodb.com
 .. _MIT License: https://github.com/electerious/Ackee/blob/master/LICENSE
 
 ----
```

Comment:
An update section is missing.

Reply:
I have not yet made an update to Ackee, so I haven't done an update yet. Maybe you can add it?

Reply:
Stopping the service and running git pull should work.